### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by gtelzur** [Codiad/Codiad#32]
I installed Fluidbyte version Fluidbyte-Codiad-cf5e0f8 on my Windows7 64bit laptop with EasyPHP version 12.1.

I changed the permissions to full control on config.php and over /data and /workspace and started index.php, then I defined a user, password and time zone with no error messages.
When I created a project i couldn't see any further actions I did. For example I created folders and uploaded files which do exist in the /workspace folder but nothing was visible in the browser (I tried it on chrome and firefox).
To sum up I can't see any files or folders below the project icon and I can't edit any file.
I would appreciate any help.

Regards,

Guy
